test/easyconfigs/easyconfigs.py

```diff
@@ -616,21 +617,26 @@ def check_python_packages(self, changed_ecs, added_ecs_filenames):
 
         failing_checks = []
 
+        python_default_urls = PythonPackage.extra_options()['source_urls'][0]
+
         for ec in changed_ecs:
 
-            ec_fn = os.path.basename(ec.path)
-            easyblock = ec.get('easyblock')
-            exts_defaultclass = ec.get('exts_defaultclass')
-            exts_default_options = ec.get('exts_default_options', {})
+            with disable_templating(ec):
+                ec_fn = os.path.basename(ec.path)
+                easyblock = ec.get('easyblock')
+                exts_defaultclass = ec.get('exts_defaultclass')
+                exts_default_options = ec.get('exts_default_options', {})
 
-            download_dep_fail = ec.get('download_dep_fail')
-            exts_download_dep_fail = ec.get('exts_download_dep_fail')
-            use_pip = ec.get('use_pip') or exts_default_options.get('use_pip')
+                download_dep_fail = ec.get('download_dep_fail')
+                exts_download_dep_fail = ec.get('exts_download_dep_fail')
+                use_pip = ec.get('use_pip', default=exts_default_options.get('use_pip'))
 
             # download_dep_fail should be set when using PythonPackage
             if easyblock == 'PythonPackage':
                 if download_dep_fail is None:
                     failing_checks.append("'download_dep_fail' should be set in %s" % ec_fn)
+                if ec.get('source_urls', resolve=False) == python_default_urls:
+                    failing_checks.append("'source_urls' can be omitted when using the default value in %s" % ec_fn)
```

Comment:
`cam be omitted` -> `should be omitted` or `should not be defined` (latter is clearer imho)